lib/views/git-tab-header-view.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import path from 'path';
+import {AuthorPropType} from '../prop-types';
+
+export default class GitTabHeaderView extends React.Component {
+  static propTypes = {
+    committer: AuthorPropType.isRequired,
+
+    // Workspace
+    workdir: PropTypes.string,
+    workdirs: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.string),
+      PropTypes.shape({
+        [Symbol.iterator]: PropTypes.func.isRequired,
+      }),
```

Comment:
I will attempt to remove and check for warnings, I did this because I am passing a `MapIterator` gathered from map.keys or map.values can't remember which i used.

Reply:
tests were using arrays, I built an extremely small helping function to make them iterators in their test files which makes only the iterator required. I realize now though I could have just sent the map iterator to arrays, but this way works too.